swap_meet/item.py

```diff
@@ -1,2 +1,20 @@
 class Item:
-    pass
+    def __init__(self, category=None, condition=0):
+        if not category:
+            category = ""
+        self.category = category
+        self.condition = condition
+
+    def condition_description(self):
+        descriptions = [
+            "horrible",
+            "really really bad",
+            "really bad",
+            "not okay",
+            "okay",
+            "fine",
+        ]
+        return descriptions[round(self.condition)]
```

Comment:
Nice use of an array for handling conditions.

I like using `round` to make sure that the condition will be an `int`!